assets/css/_route_picker.scss

```diff
@@ -24,6 +24,71 @@
   overflow-y: scroll;
 }
 
+.m-route-filter__type {
```

Comment:
With a `routeFilter.tsx` more isolated, this should definitely be in its own `_route_filter.scss`